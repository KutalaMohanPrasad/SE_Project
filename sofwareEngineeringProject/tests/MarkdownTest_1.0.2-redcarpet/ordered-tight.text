## Ordered

Tight:

1.	First
2.	Second
3.	Third

and:

1. One
2. Two
3. Three
