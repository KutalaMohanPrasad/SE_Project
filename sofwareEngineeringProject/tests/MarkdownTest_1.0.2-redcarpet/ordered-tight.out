<h2>Ordered</h2>

<p>Tight:</p>

<ol>
<li>First</li>
<li>Second</li>
<li>Third</li>
</ol>

<p>and:</p>

<ol>
<li>One</li>
<li>Two</li>
<li>Three</li>
</ol>